### DISCUSSION TITLE
Enabling DWIN support for games on Ender 3 V2

### DISCUSSION THREAD

**Discussion by tiagofreire-pt** [Jyers/Marlin#747] [Ideas]
The Marlin vanila has 4 games included, but seems to have support only for non-DWIN screens.

Supporting this should be merely a screen menu option that is not available/coded on this screen firmware.

`Configuration_adv.h`

```
  // Frivolous Game Options
  #define MARLIN_BRICKOUT
  #define MARLIN_INVADERS
  #define MARLIN_SNAKE
  #define GAMES_EASTER_EGG   
```

**Comment by tonyG433** [comment]
Does that mean i can beat snake on a 3d printer? Seems like a great idea :)

**Reply by tiagofreire-pt** [reply]
Yes, just for fighting boredom. 💯

**Comment by Jyers** [comment]
> Supporting this should be merely a screen menu option that is not available/coded on this screen firmware.

Ha, I wish.... This would take an extensive amount of work to implement and is therefore in no way a priority. Maybe if I run out of feature ideas and get bored I'll take a crack at it some time.
